arb_os/arbosTest.mini

```diff
@@ -39,20 +39,19 @@ use evmCallStack::evmCallFrame_getAccount;
 
 public impure func arbosTest_txcall() {
     if let Some(topFrame) = evmCallStack_topFrame() {
-        if (topFrame.caller != address(0)) {
-            evmOp_revert_knownCodePc(address(const::Address_ArbosTest), 0, 0, 0);
-        }
```

Comment:
Don't we want to keep this protection, or are the dangerous methods protected elsewhere?

Reply:
They're protected elsewhere. Every dangerous method verifies that it is being called by address zero (because they're meant only for testing).